addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -331,11 +333,11 @@ var WysiwygMultizone = Wysiwyg.extend({
         var data = this._super();
         var res_id = $editable.data('oe-id');
         var res_model = $editable.data('oe-model');
-        if (!$editable.data('oe-model')) {
-            var object = $('html').data('main-object');
-            res_model = object.split('(')[0];
-            res_id = +object.split('(')[1].split(',')[0];
-        }
+        // if (!$editable.data('oe-model')) {       // TEMP COMMENT - Error JS in restricted editor
+        //     var object = $('html').data('main-object');
+        //     res_model = object.split('(')[0];
+        //     res_id = +object.split('(')[1].split(',')[0];
+        // }
```

Comment:
@qsm-odoo In master and saas-12.2, Restricted Editor does not work. When trying to click on Edit button, there is a JS error from the lines above. I've commented these lines (and then it works) to be able to work and test my task. This said, I don't really know what to do and what is the use of these lines. It seems to come from this commit https://github.com/odoo/odoo/pull/29775/commits/02b5f184b9209df41ae99415056ca52e801e4d0c. Are you aware of this ? Should I ask directly to the author ?

Reply:
I reported the problem to the editor team.